src/authorization.js

```diff
@@ -415,16 +413,16 @@ class Authorization {
     if (this.virtual) {
       return []
     }
-    var statement
-    var fragment = rdf.namedNode('#' + this.hashFragment())
-    var ns = vocab(rdf)
-    var statements = [
+    let statement
+    let fragment = rdf.namedNode('#' + this.hashFragment())
```

Comment:
if single Authorization doesn't have to represent itself as RDF graph, possibly code from `Authorization#rdfStatements()` could move to `PermissionSet#buildGraph()` which would not require passing `aclUrl` to constructor of `Authorization`